web/src/js/components/MainView.tsx

```diff
@@ -11,6 +11,9 @@ export default function MainView() {
     const hasSelection = useAppSelector(
         (state) => !!state.flows.byId[state.flows.selected[0]],
     );
+    const isMultipleFlowsSelection = useAppSelector(
+        (state) => state.flows.selected.length > 1,
+    );
```

Comment:
```suggestion
    const hasFlowSelected = useAppSelector(
        (state) => state.flows.selected.length === 1 && !!state.flows.byId[state.flows.selected[0]],
    );
```

The primary benefit of this change is that you will need to address the other usage of `hasSelection` and fix it in the process. No need for a splitter if there's no flow view 😉